pkg/hostagent/services.go

```diff
@@ -522,19 +513,19 @@ func (agent *HostAgent) getInfrastucreIp(ipType string) string {
 	}
 	err = rclient.Get(context.TODO(), types.NamespacedName{
 		Name: "cluster"}, infraStructureInfo)
-	agent.log.Debug("infraStructureInfo####: ", infraStructureInfo, err)
 	if err != nil {
 		return ""
 	}
 	if infraStructureInfo.Status.Platform == configv1.OpenStackPlatformType {
 		if infraStructureInfo.Status.PlatformStatus != nil &&
 			infraStructureInfo.Status.PlatformStatus.OpenStack != nil {
-			agent.log.Debug("IngressIP: ", infraStructureInfo.Status.PlatformStatus.OpenStack.IngressIP)
-			switch ipType {
-			case IngressIp:
+			switch serviceName {
+			case RouterInternalDefult:
```

Comment:
There is a typo in this name, but I think its consistently wrong so I guess we might be okay!